deps/rabbit/src/rabbit_peer_discovery.erl

```diff
@@ -82,6 +85,16 @@ node_type() ->
             ?DEFAULT_NODE_TYPE
     end.
 
+-spec registration() -> true | false.
```

Comment:
You can use `boolean()` instead of `true | false`. That’s the builtin type spec for this.